modules/ROOT/pages/migrate-ah.adoc

```diff
@@ -0,0 +1,44 @@
+:experimental:
+
+= Migrating from Atomic Host (AH) to Fedora CoreOS (FCOS)
+
+== Overview
+
+https://www.projectatomic.io/[Atomic Host] is a system to deploy applications in containers. Atomic Hosts users are encouraged to migrate to FCOS, as the project is being folded into FCOS.
+
+Atomic Host uses a cloud-config file to define parameters of the containers. There is currently no tool to migrate cloud-config files to FCOS Ignition files. The conversion is not direct, as some parameters in Ignition are located in different Atomic Host files.
```

Comment:
- A little more info into the first sentence:
    - "Fedora Atomic Host used `cloud-init` for provisioning, which requires users to provide a cloud-config file as userdata for instance configuration."
- For the second sentence maybe something a little more like:
    - Since Ignition and cloud-init have overlapping but different feature sets the conversion from `cloud-init` cloud-config files to Ignition configs is non-trivial. No tool exists to do this today."